### DISCUSSION TITLE
PasswordInput - tabIndex = -1 on the rightSection?

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#131] [Q&A] [answered]
This may not be desirable for usability reasons - just thought I'd raise it.

Should the `rightSection` button have a negative tabIndex?
It's generally not a field you want to tab to and if you're using a screen-reader then I am sure it's handling that field in a special way anyway. (so the visibility toggle is maybe not helpful?)

I've just noticed that in testing on a password change form, it is intuitive to tab between the two fields (old password & new password), but with that button in the middle it's an extra tab press.

**Comment by rtivital** [comment, marked as answer]
Yep, that's a good point, I'll add an option to change tabIndex of toggle button in 2.0.2, with default to -1

**Reply by cstrat** [reply]
Fantastic thanks!
